test/unit/test_bundling.js

```diff
@@ -1,5 +1,5 @@
-/* global describe, it, beforeEach, afterEach */
 "use strict";
+const { describe, it, beforeEach, afterEach } = require("node:test");
 
```

Comment:
For consistency:

```suggestion

let { describe, it, beforeEach, afterEach } = require("node:test");
```

In fact, strictly speaking, this line should be moved a little further down: Just above `require("path")` (because local > third-party > stdlib).

Reply:
Fix uped :heavy_check_mark: